### PR TITLE
Remove missing files from the file list.

### PR DESCRIPTION
**Pull request by xhochy**
These files are only really missing if the user executed `conda clean -p`.

### CODE REVIEW

**Comment by mcg1969** [comment]
@xhochy can you revisit this one? I'm afraid there are some nontrivial merge conflicts now. In particular, note that I created a `setup_envs.bat` for Windows that mirrors `setup_envs.sh` for Unix because I was having issues with bash on Windows.

**Comment by mcg1969** [comment]
Now that we're using GitHub Actions for CI you can just delete `.travis.yml`. And `setup_envs.bat` is gone again, too; we're now back down to just `setup_envs.sh`. So it should be easier to merge your changes.

**Comment by mcg1969** [comment]
Now that I've spent so much time wading in this, I'm actually going to propose the following: that conda pack should always be tested against _its own conda installation_—or more specifically, its own package cache—so that it's free to manipulate that cache however it wants to without distrupting the user's local installation.

That's effectively what I do in the GitHub Action. For the raw package build and the doc build, I'm using the worker's built-in Miniconda install. But for the testing, I'm installing our own conda environment.

**Comment by mcg1969** [comment]
It doesn't even matter, actually, if the `conda` used to create the environment is the same as the `conda` used to run the test! Because the code determines the location of the cached packages by the metadata in the environment.

**Comment by mcg1969** [comment]
OK, with https://github.com/conda/conda-pack/pull/135 I've changed the testbed in a significant way: it now creates a full `conda` tree, including a root environment with `conda` installed, and with _that_ `conda` installing the individual test environments. This is significant because it enables the testbed to _control its own package cache_ without disrupting any other work the developer might be doing.

So with this, we can free free to remove package cache entries at will, as we now do with the Python 2.7 version of the `conda_pack_test_lib2` package.

**Comment by xhochy** [comment]
Due to the merge of a very old branch, this has triggered `darker` for a lot of the existing codebase. Thus most of the changes are unrelated. I hope it is still reviewable.

File: testing/setup_envs.sh [hunk 1]
```diff
@@ -4,8 +4,22 @@ set -eo pipefail
...
+conda env create -f "${current_dir}/env_yamls/py36.yml" -p "${current_dir}/environments/py36_missing_files" $@
+if [[ "$OS" == "Windows_NT" ]]; then
+  rm "${current_dir}/environments/py36_missing_files/Lib/site-packages/toolz/__init__.py"
+else
+  rm "${current_dir}/environments/py36_missing_files/lib/python3.6/site-packages/toolz/__init__.py"
+fi
+# Only do this when the developer has agreed to it, this might otherwise break things in his system.
+if [[ "$CONDA_CLEAN_P" == "1" ]]; then
+  conda clean -apfy
+fi
```
Comment (jaimergp) [inline testing/setup_envs.sh]: This is a bit cryptic. Can we use a word here instead of `"1"`?

**Review by jaimergp — APPROVED** [review]

**Pull request merged**